code/cfilearchiver/cfilearchiver.cpp

```diff
@@ -318,24 +318,6 @@ void print_instructions()
 	printf( "Creates a vp archive out of a FreeSpace data tree.\n\n" );
 	printf( "Usage:     cfilearchiver archive_name src_dir\n");
 	printf( "Example:   cfilearchiver freespace /tmp/freespace/data\n\n");
-	printf( "Directory structure options:\n" );
```

Comment:
I think the only alternative that makes sense is to link to http://www.hard-light.net/wiki/index.php/FS2_Data_Structure
After all, that's a much better explanation of what file is expected where than what can or should be expressed in a commandline output.